error[E0658]: `#[diagnostic]` attribute name space is experimental
   --> .../host/share/cargo/registry/src/index.crates.io-*/axum-0.7.5/src/handler/mod.rs:130:5
    |
130 |     diagnostic::on_unimplemented(
    |     ^^^^^^^^^^
    |
    = note: see issue #111996 <https://github.com/rust-lang/rust/issues/111996> for more information
    = help: add `#![feature(diagnostic_namespace)]` to the crate attributes to enable
    = note: this compiler was built on 2024-04-09; consider upgrading it if it is out of date

For more information about this error, try `rustc --explain E0658`.
error: could not compile `axum` (lib) due to 1 previous error

---

--- a/crates/atuin-server/Cargo.toml
+++ b/crates/atuin-server/Cargo.toml
@@ -25,7 +25,7 @@ base64 = { workspace = true }
 rand = { workspace = true }
 tokio = { workspace = true }
 async-trait = { workspace = true }
-axum = "0.7.4"
+axum = "=0.7.4"
 axum-server = { version = "0.6.0", features = ["tls-rustls"] }
 fs-err = { workspace = true }
 tower = { workspace = true }
